Gemfile.lock

```diff
@@ -0,0 +1,86 @@
+GIT
+  remote: https://github.com/pmd/pmd-regression-tester.git
+  revision: 8e652477b3547d6c843864a2641389a4c61c383e
+  specs:
+    pmdtester (1.0.0.pre.SNAPSHOT)
+      differ
+      nokogiri (~> 1.8.2)
+      rufus-scheduler (~> 3.5, >= 3.5)
+      slop (~> 4.6.2)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.7.0)
+      public_suffix (>= 2.0.2, < 5.0)
+    claide (1.0.3)
+    claide-plugins (0.9.2)
+      cork
+      nap
+      open4 (~> 1.3)
+    colored2 (3.1.2)
+    concurrent-ruby (1.1.5)
+    cork (0.3.0)
+      colored2 (~> 3.1)
+    danger (5.16.1)
+      claide (~> 1.0)
+      claide-plugins (>= 0.9.2)
+      colored2 (~> 3.1)
+      cork (~> 0.1)
+      faraday (~> 0.9)
+      faraday-http-cache (~> 1.0)
+      git (~> 1.5)
+      kramdown (~> 1.5)
+      no_proxy_fix
+      octokit (~> 4.7)
+      terminal-table (~> 1)
+    differ (0.1.2)
+    et-orbi (1.2.2)
+      tzinfo
+    faraday (0.16.2)
```

Comment:
Will this really work? It looks like it's going to take the newer faraday version

Reply:
Yes, 0.16.2 is the fixed version of faraday (0.16.0 and 0.16.1 were broken).

Reply:
Ok thanks